src/test/java/bio/terra/service/job/JobServiceTest.java

```diff
@@ -157,18 +158,21 @@ public void enumerateJobsPermissionTest() throws Exception {
     // The fids list should be in exactly the same order as the database ordered by submit time.
 
     // Launch 2 jobs that are not visible to the test user
+    List<String> privateJobIds = new ArrayList<>();
     UUID privateDatasetId = UUID.randomUUID();
     for (int i = 0; i < 2; i++) {
-      runFlightWithParameters(
-          makeDescription(i), makeFlightClass(i), testUser, String.valueOf(privateDatasetId));
+      String jobId =
+          runFlightWithParameters(
+              makeDescription(i), makeFlightClass(i), testUser, String.valueOf(privateDatasetId));
+      privateJobIds.add(jobId);
     }
     when(samService.listActions(
             testUser2, IamResourceType.DATASET, String.valueOf(privateDatasetId)))
         .thenReturn(List.of());
 
     assertEquals(
         "no jobs are visible",
-        jobService.enumerateJobs(0, 100, testUser2, SqlSortDirection.ASC, "").size(),
+        jobService.enumerateJobs(0, 100, testUser2, SqlSortDirection.ASC, "", null).size(),
```

Comment:
If null `jobIds` would no longer be passed to `jobService.enumerateJobs`, I would adjust the calls in tests as well.

Reply:
I agree, although for the test code I might make a helper function that handles all the common cases, it looks like most calls have the same last three parameters. E.g.,
```
  private List<JobModel> enumerateJobs(int offset, int limit) {
    return enumerateJobs(offset, limit, testUser, SqlSortDirection.ASC, "");
  }
```